packages/core/src/app/ovm/quantifiers/merkle-inclusion-quantifier.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  QuantifierResult,
+  Quantifier,
+  BigNumber,
+  MerkleTree,
+} from '../../../types'
+
+interface MerkleInclusionQuantifierParameters {
+  root: Buffer
+  key: BigNumber
+}
+
+/*
+ * The MerkleInclusionQuantifier that quantifies data at a specific key of a Merkle Tree,
+ * if data exists at the key.
+ */
+export class MerkleInclusionQuantifier implements Quantifier {
+  public constructor(private readonly tree: MerkleTree) {}
```

Comment:
So it's interesting that we pass a tree into the constructor, but then also we query the `root`--as intuitively "a merkle tree' has just one unique root.  I guess this is because the `MerkleTree` values can be updated.  I wonder if we should specify that the tree class is dynamic?

Reply:
The `MerkleTree` interface specifies that itself, no?

Reply:
Oh totally--I guess I was just saying we could consider naming the interface `UpdateableMerkleTree` or something of the like.  Not sure it's worth the verbosity though, you're right that that is apparent when you check the interface (which is pretty concise)